.github/workflows/testing.yml

```diff
@@ -1,6 +1,7 @@
 name: Testing
 
 on:
+  workflow_dispatch:
```

Comment:
I do not think this trigger is required here - it makes sense for parametrised builds, but there are not parameters here.